tests/test_graphql_api/test_application_round/test_set_handled.py

```diff
@@ -124,3 +132,36 @@ def test_application_round__set_handled__unit_admin__has_perms_to_all_units(grap
 
     application_round.refresh_from_db()
     assert application_round.status == ApplicationRoundStatusChoice.HANDLED
+
+
+@pytest.mark.parametrize(
+    ("application_status", "raises_error"),
+    [
+        (ApplicationStatusChoice.DRAFT, False),
```

Comment:
`DRAFT` applications should be automatically `EXPIRED` if `ApplicationRound` is in the `IN_ALLOCATION` stage. The test still works, but I guess this is redundant.